colibre/auto_plotter/metallicity.yml

```diff
@@ -37,8 +37,6 @@ stellar_mass_gas_sf_metallicity_fromz_lom_30:
     - filename: GalaxyStellarMassGasMetallicity/Fraser-McKelvie_2021.hdf5
     - filename: GalaxyStellarMassGasMetallicity/FIREbox.hdf5
     - filename: GalaxyStellarMassGasMetallicity/Strom2022_OH.hdf5
-    - filename: GalaxyStellarMassGasMetallicity/Curti2020.hdf5
-    - filename: GalaxyStellarMassGasMetallicity/Lewis2023.hdf5
```

Comment:
@correac Is removing Curti+2020 and Lewis+2023 data done on purpose? (I don't see that you have commented on this change in the description of this PR)

Note that there are also many other changes in the same file similar to this one.

Reply:
No, removing that data was not intended. Let me check what happened..